rtsyntax.php

```diff
@@ -72,58 +112,113 @@ public function settings_field() {
-    }
-
-    public function admin_page() { ?>
-        <div class="wrap">
-            <h2><?php _e( 'rtSyntax', 'rtSyntax' ); ?></h2>
-            <form method="post" action="options.php">
-                <?php settings_fields( 'rtsyntax' ); ?>
-                <?php do_settings_sections( 'rtsyntax' ); ?>
-                <?php do_settings_fields( 'rtsyntax', 'rtsyntax-theme' ); ?>
-                <?php submit_button(); ?>
-            </form>
-        </div><?php
-    }
-
-    public function rtsyntax_buttons( $plugin_array ) {
-        $plugin_array['rtsyntax'] = plugin_dir_url(__FILE__).'js/rtsyntax.js';
-        $plugin_array['rtcode'] = plugin_dir_url(__FILE__).'js/rtsyntax.js';
-        $plugin_array['rtkey'] = plugin_dir_url(__FILE__).'js/rtsyntax.js';
-        return $plugin_array;
-    }
-
-    public function register_rtsyntax_buttons( $buttons ) {
-        array_push( $buttons, "separator", "rtsyntax", "rtcode", "rtkey", "code" );
-        return $buttons;
-    }
-
-    public function enqueue() {
-        $options = get_option( 'rtsyntax_options' );
-        wp_enqueue_style( 'rtsyntax-' . $options['theme'], plugin_dir_url(__FILE__) . 'css/' . $options['theme'] . '.css' );
-        wp_enqueue_script( 'rtsyntax', plugin_dir_url(__FILE__) . 'js/highlight.js', array(), null, true );
-    }
-
-    public function onload() { ?>
-        <script>
-            jQuery(function ($) {
-                if( typeof hljs === 'object' ) {
-                    hljs.initHighlightingOnLoad();
-                }
-            } );
-        </script><?php
-    }
-
-    public function convert_pres( $content ){
-        $content = str_replace( '<pre>', '<pre class="no-highlight">', $content );
-        return preg_replace( '/<pre(.*)>(.*)<\/pre>/isU', '<pre><code$1>$2</code></pre>', $content );
-    }
+        </select>
+		<?php
+	}
+
+	/**
+	 * Add settings page on admin side
+	 */
+	public function admin() {
+		add_options_page( __( 'rtSyntax', 'rtSyntax' ), __( 'rtSyntax', 'rtSyntax' ), 'manage_options', 'rtsyntax', array( $this, 'admin_page' ) );
+	}
+
+	/**
+	 * Add section for registered settings
+	 */
+	public function admin_page() {
+	?>
+		<div class="wrap">
+		<h2><?php esc_html_e( 'rtSyntax', 'rtSyntax' ); ?></h2>
+		<form method="post" action="options.php">
+			<?php settings_fields( 'rtsyntax' ); ?>
+			<?php do_settings_sections( 'rtsyntax' ); ?>
+			<?php do_settings_fields( 'rtsyntax', 'rtsyntax-theme' ); ?>
+			<?php submit_button(); ?>
+		</form>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Register buttons to display on tinyMCE editor
+	 *
+	 * @param Array $plugin_array Pulugin Related Array.
+	 *
+	 * @return mixed
+	 */
+	public function rtsyntax_buttons( $plugin_array ) {
+		$plugin_array['rtsyntax'] = plugin_dir_url( __FILE__ ) . 'js/rtsyntax.js';
+		$plugin_array['rtcode']   = plugin_dir_url( __FILE__ ) . 'js/rtsyntax.js';
+		$plugin_array['rtkey']    = plugin_dir_url( __FILE__ ) . 'js/rtsyntax.js';
```

Comment:
Define constant `RTSYNTAX_DIR_URL` and use it here instead of calling `plugin_dir_url( __FILE__ )` every time.